replibyte/src/source/mysql.rs

```diff
@@ -0,0 +1,256 @@
+use std::collections::HashMap;
+use std::io::{BufReader, Error, ErrorKind};
+use std::process::{Command, Stdio};
+
+use dump_parser::postgres::{
```

Comment:
We may want to abstract some of this to have to use from `...::postgres::..`.

Reply:
agree

Reply:
Could you explain more? Is that something I should fix now or for the future? @pepoviola